app/models/dataset_record/darwin_core/occurrence.rb

```diff
@@ -953,11 +953,17 @@ def parse_typestatus(type_status, taxon_protonym)
           type_type: type_status_parsed[:type].downcase
         }
       end
-      if (original_combination_protonym = Protonym.find_by(cached_original_combination: type_scientific_name, project_id: self.project_id))
+      original_combination_protonyms = Protonym.where(cached_original_combination: type_scientific_name, project_id: self.project_id)
+      if original_combination_protonyms.count == 1
         return {
           type_type: type_status_parsed[:type].downcase,
-          protonym: original_combination_protonym
+          protonym: original_combination_protonyms.first
         }
+      elsif original_combination_protonyms.count > 1
+        potential_protonym_strings = original_combination_protonyms.map { |proto| "[id: #{proto.id} #{proto.cached_original_combination_html}]" }.join(', ')
```

Comment:
Does it actually render as HTML or thinks like `<i>Jivarus alienus</i>` are displayed literal instead?

Reply:
We should not reference _html version, use the non in all comparisons.

Reply:
It renders the HTML in the error message:
<img width="314" alt="image" src="https://github.com/SpeciesFileGroup/taxonworks/assets/33206210/36d4e7d1-8b13-4617-99c0-3a98652ed31e">